ocean_lib/ocean/ocean_assets.py

```diff
@@ -64,6 +66,7 @@ def _process_service_descriptors(
         metadata: dict,
         provider_uri: str,
         wallet: Wallet,
+        trusted_algorithms: list,
```

Comment:
I just noticed this! I remember coding this myself in the experimental commit, but it was probably overwritten in the meantime. The default value should be None here (and probably no type hinting) See: https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/